mx.py

```diff
@@ -9856,7 +9856,7 @@ def log(msg=None):
     if msg is None:
         print
     else:
-        print msg
```

Comment:
Won't this just cause a lot of empty lines? Or does the `print` statement in python not add a `\n` if there's already one in `msg`?

Reply:
That kind of python magic definitely deserves a comment. Can you please add your above description as a comment.